docs/tempo/website/configuration/_index.md

```diff
@@ -10,15 +10,14 @@ This section explains the configuration options for Tempo as well as the details
   - [Authentication/Server](#authenticationserver)
   - [Distributor](#distributor)
   - [Ingester](#ingester)
-  - [Query Frontend](#queryfrontend)
+  - [Query Frontend](#query-frontend)
   - [Querier](#querier)
   - [Compactor](#compactor)
   - [Storage](#storage)
   - [Memberlist](#memberlist)
-  - [Compression](#compression)
```

Comment:
This section was added by https://github.com/grafana/tempo/commit/3b3b9e2c5e93ad59144a8b6c5039fc6f982256a0. But contrary to the other sections, this doesn't refer to a header within this page and should link to a different page (which is listed in the sidebar already).

People should be able to discover the page using the sidebar, so I think we can remove this.